mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -693,6 +695,7 @@ class ALSSuite extends MLTest with DefaultReadWriteTest with Logging {
       }.getCause.getMessage.contains(msg))
     }
     withClue("transform should fail when ids exceed integer range. ") {
+      spark.conf.set(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "false")
```

Comment:
how does this test fail if we enable AQE?

Reply:
The same question with @cloud-fan 's. We need a comment about the reason of exclusion.